index.html

```diff
@@ -10,30 +10,62 @@
   </head>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/234526696-d7ec7b2e-1658-4985-ae4b-b44cf3158b7e.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalized 👍 because some of your commit messages are not capitalized 😨 Please consider this issue to make your pull request professional and descriptive ☺️